server/controllers/admin/enterprises.js

```diff
@@ -0,0 +1,76 @@
+/**
+* Enterprises Controller
+*
+* This controller is responsible for creating and updating Enterprises.  Every
+*/
+var db = require('../../lib/db');
+
+
+exports.list = function list(req, res, next) {
+  'use strict';
+
+  var sql;
+
+  sql =
+    'SELECT id, name, abbr, email, po_box, phone, location_id, logo, currency_id FROM enterprise';
```

Comment:
In general, the list method should return a minimum number of elements to prevent sending lots of unnecessary data to the client.  Try this API:
 1. GET /enterprises  --> returns an array of JSONs like this: `{ id: [number], name : [string], abbr: [string] }`
 2. GET /enterprises?detailed=1  --> returns an array of JSONs with more details (id, name, abbr, email, po_box, phone, location_id, logo, currency_id).
 3. GET /enterprises/:id  --> returns a single JSON with properties (id, name, abbr, email, po_box, phone, location_id, logo, currency_id).